### PR TITLE
Translations - Add Polish Translations

### PR DESCRIPTION
**Pull request by Krzyciu**
**When merged this pull request will:**
- Add missing polish translations
- Remove russian translation from polish - ace_common

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/fieldmanual/stringtable.xml [hunk 1]
```diff
@@ -105,171 +128,231 @@
...
+            <Polish>%3DAGR%4 jest uproszczoną wersją%3MicroDAGR GPS%4. Posiada podobne funkcje, ale brakuje mu map dostępnych w %3MicroDAGR GPS%4.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Wyekwipuj %3DAGR%4.&lt;br/&gt;%2Użyj [%3%12%4] i wybierz %3Konfiguruj%4 lub %3Przełącz%4.&lt;br/&gt;&lt;br/&gt;Poniższe menu są dostępne podczas konfiguracji %3DAGR:%4&lt;br/&gt;%11%2Widok Danych: WIP&lt;br/&gt;%11%2GoTo WP: Wybierz waypoint do śledzenia.&lt;br/&gt;%11%2WP List: Dodaj/Edytuj/Usuń waypointy.&lt;br/&gt;%11%2Połącz do: Połącz %3DAGR%4 do dalmierza %3Vector 21%4.&lt;br/&gt;%11%2Opcje
+</Polish>
```
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:192]:
```suggestion
            </Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 2]
```diff
@@ -105,171 +128,231 @@
...
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_DeadManSwitch_ShortName">
             <English>Explosive Revenge</English>
+            <Polish>Wybuchowa Zemsta</Polish>
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_DeadManSwitch_Description">
             <English>The %3Dead Man's Switch%4 is a device that allows a soldier to detonate an %3Explosive%4 when the soldier dies.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;%2Use [%3%12%4] and select %3Explosives%4.&lt;br/&gt;%2Select %3Dead Man's Switch%4 and connect the desired %3Explosive%4.&lt;br/&gt;%2Repeat the process and disconnect to reverse.</English>
+            <Polish>%3Czuwak%4 jest urządzeniem pozwalającym wysadzić %3Ładunek Wybuchowy%4 gdy żołnierz zginie.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Użyj [%3%12%4] i wybierz %3Mat. Wybuchowe%4.&lt;br/&gt;%2Wybierz %3Czuwak%4 i połącz wybrany %3Ładunek Wybuchowy%4.&lt;br/&gt;%2W celu odłączenia powtórz czynności i wybierz odłącz.
+</Polish>
```
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:201]:
```suggestion
            </Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 3]
```diff
@@ -105,171 +128,231 @@
...
+            <Polish>%3Liny%4 mają wiele zastosowań, takich jak %3Holowanie%4 pojazdów czy %3Zjeżdżanie na Linach%4 z helikopterów.&lt;br/&gt;&lt;br/&gt;%3Holowanie:%4&lt;br/&gt;%2Podejdź pod pojazd.&lt;br/&gt;%2Użyj [%3%13%4] i wybierz %3Holowanie%4.&lt;br/&gt;%2Wybierz długość liny. &lt;br/&gt;%2Wybierz punkt zaczepu liny na holującym pojeździe.&lt;br/&gt;%2Wybierz punkt zaczepu liny na holowanym pojeździe.&lt;br/&gt;&lt;br/&gt;%3Dostępne Długości Liny:%4&lt;br/&gt;%2 3.2 metera&lt;br/&gt;%2 6.2 metera&lt;br/&gt;%2 12.2 metera&lt;br/&gt;%2 15.2 metera&lt;br/&gt;%2 18.3 metera&lt;br/&gt;%2 27.4 metera&lt;br/&gt;%2 36.6 metera
+
+Przetłumaczono z www.DeepL.com/Translator (wersja darmowa)</Polish>
```
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:303]: not sure about the url?
```suggestion
Przetłumaczono z www.DeepL.com/Translator (wersja darmowa)
            </Polish>
```

File: addons/fieldmanual/stringtable.xml (hunk 1 above)
Comment (PabstMirror) [reply addons/fieldmanual/stringtable.xml:192]: oh wait, these should be on the same line

File: addons/fieldmanual/stringtable.xml (hunk 3 above)
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:303]:
```suggestion
            <Polish>%3Liny%4 mają wiele zastosowań, takich jak %3Holowanie%4 pojazdów czy %3Zjeżdżanie na Linach%4 z helikopterów.&lt;br/&gt;&lt;br/&gt;%3Holowanie:%4&lt;br/&gt;%2Podejdź pod pojazd.&lt;br/&gt;%2Użyj [%3%13%4] i wybierz %3Holowanie%4.&lt;br/&gt;%2Wybierz długość liny. &lt;br/&gt;%2Wybierz punkt zaczepu liny na holującym pojeździe.&lt;br/&gt;%2Wybierz punkt zaczepu liny na holowanym pojeździe.&lt;br/&gt;&lt;br/&gt;%3Dostępne Długości Liny:%4&lt;br/&gt;%2 3.2 metera&lt;br/&gt;%2 6.2 metera&lt;br/&gt;%2 12.2 metera&lt;br/&gt;%2 15.2 metera&lt;br/&gt;%2 18.3 metera&lt;br/&gt;%2 27.4 metera&lt;br/&gt;%2 36.6 metera Przetłumaczono z www.DeepL.com/Translator (wersja darmowa)</Polish>
```
not sure about the url at end?

File: addons/fieldmanual/stringtable.xml [hunk 4]
```diff
@@ -289,27 +372,35 @@
         </Key>
         <Key ID="STR_ACE_FieldManual_HintCategory_FieldManual_DisplayName">
             <English>ACE3</English>
+            <Polish>ACE3</Polish>
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_FortifyTool_ShortName">
             <English>Build Fortifications</English>
+            <Polish>Budowanie  Fortyfikacji</Polish>
```
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:379]:
```suggestion
            <Polish>Budowanie Fortyfikacji</Polish>
```

File: addons/fieldmanual/stringtable.xml (hunk 2 above)
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:201]:
```suggestion
            <Polish>%3Czuwak%4 jest urządzeniem pozwalającym wysadzić %3Ładunek Wybuchowy%4 gdy żołnierz zginie.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Użyj [%3%12%4] i wybierz %3Mat. Wybuchowe%4.&lt;br/&gt;%2Wybierz %3Czuwak%4 i połącz wybrany %3Ładunek Wybuchowy%4.&lt;br/&gt;%2W celu odłączenia powtórz czynności i wybierz odłącz.</Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 5]
```diff
@@ -105,171 +128,228 @@
...
+            <Polish>%3DAGR%4 jest uproszczoną wersją%3MicroDAGR GPS%4. Posiada podobne funkcje, ale brakuje mu map dostępnych w %3MicroDAGR GPS%4.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Wyekwipuj %3DAGR%4.&lt;br/&gt;%2Użyj [%3%12%4] i wybierz %3Konfiguruj%4 lub %3Przełącz%4.&lt;br/&gt;&lt;br/&gt;Poniższe menu są dostępne podczas konfiguracji %3DAGR:%4&lt;br/&gt;%11%2Widok Danych: WIP&lt;br/&gt;%11%2GoTo WP: Wybierz waypoint do śledzenia.&lt;br/&gt;%11%2WP List: Dodaj/Edytuj/Usuń waypointy.&lt;br/&gt;%11%2Połącz do: Połącz %3DAGR%4 do dalmierza %3Vector 21%4.&lt;br/&gt;%11%2Opcje
+</Polish>
```
Comment (PabstMirror) [inline addons/fieldmanual/stringtable.xml:192]:
```suggestion
            <Polish>%3DAGR%4 jest uproszczoną wersją%3MicroDAGR GPS%4. Posiada podobne funkcje, ale brakuje mu map dostępnych w %3MicroDAGR GPS%4.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Wyekwipuj %3DAGR%4.&lt;br/&gt;%2Użyj [%3%12%4] i wybierz %3Konfiguruj%4 lub %3Przełącz%4.&lt;br/&gt;&lt;br/&gt;Poniższe menu są dostępne podczas konfiguracji %3DAGR:%4&lt;br/&gt;%11%2Widok Danych: WIP&lt;br/&gt;%11%2GoTo WP: Wybierz waypoint do śledzenia.&lt;br/&gt;%11%2WP List: Dodaj/Edytuj/Usuń waypointy.&lt;br/&gt;%11%2Połącz do: Połącz %3DAGR%4 do dalmierza %3Vector 21%4.&lt;br/&gt;%11%2Opcje</Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 6]
```diff
@@ -105,171 +128,227 @@
...
             <English>The %3Horus ATragMX%4 considers atmospheric conditions, gun data, ammunition, range, speed, and muzzle velocity to calculate precise aiming solutions with %3Come-Up%4 results - and even accounts for %3Coriolis%4 and %3Spin Drift%4 effects. %3ATragMX%4, loaded on a handheld computer made by %3TDS Recon%4, is easy to use and lightning fast. The %3Recon%4 meets the rigorous %3MIL-STD-810F%4 military standard for drops, vibration, humidity, altitude and extreme temperatures.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;Please visit the wiki page for more information.</English>
+            <Polish>%3Horus ATragMX%4 uwzględnia warunki atmosferyczne, dane broni, amunicję, zasięg, prędkość wylotową do wyliczenia precyzyjnych nastaw - ponadto uwzględnia %3Efekt Coriolisa%4 oraz %3Efekt Spinowy%4. %3ATragMX%4, załadowany na komputer przenośny wyprodukowany przez %3TDS Recon%4, jest łatwym i prostym w użyciu. %3Recon%4 spełnia rygorystyczne wymagania %3MIL-STD-810F%4 pod względem odporności na upadek, wibracje, wilgotność, wyskość oraz ekstremalne temperatury.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;Wejdź na wiki po więcej informacji.</Polish>
```
Comment (veteran29) [inline addons/fieldmanual/stringtable.xml:143]:
```suggestion
            <Polish>%3Horus ATragMX%4 uwzględnia warunki atmosferyczne, dane broni, amunicję, zasięg, prędkość wylotową do wyliczenia precyzyjnych nastaw - ponadto uwzględnia %3Efekt Coriolisa%4 oraz %3Efekt Spinowy%4. %3ATragMX%4, załadowany na komputer przenośny wyprodukowany przez %3TDS Recon%4, jest łatwy i prosty w użyciu. %3Recon%4 spełnia rygorystyczne wymagania %3MIL-STD-810F%4 pod względem odporności na upadek, wibracje, wilgotność, wyskość oraz ekstremalne temperatury.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;Wejdź na wiki po więcej informacji.</Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 7]
```diff
@@ -105,171 +128,227 @@
...
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_Sandbag_Description">
             <English>%3Sandbags%4 are sacks made of sturdy material, filled with sand, used for a variety of purposes such as creating barriers or providing stability in construction projects. Useful in expanding larger placed fortifications.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;%2Equip a %3Sandbag (Empty)%4.&lt;br/&gt;%2Use [%3%12%4] and select %3Deploy Sandbag%4.&lt;br/&gt;%2Follow on-screen instructions for placement.</English>
+            <Polish>%3Worki z Piaskiem%4 to worki wykonane z mocnego materiału, wypełnione piaskiem, używane do różnych celów, takich jak tworzenie barier lub zapewnianie stabilności w projektach budowlanych. Przydatne przy tworzeniu większych fortyfikacji.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Wyekwipuj %3Worek z Piaskiem (Pusty%4.&lt;br/&gt;%2Use [%3%12%4] i wybierz %3Rozłóż Worek z Piaskiem%4.&lt;br/&gt;%2Podążaj za instrukcjami na ekranie.</Polish>
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_SpareBarrels_ShortName">
             <English>Lower Firearm Temperature</English>
+            <Polish>Niższa Temperatura twoje Broni</Polish>
```
Comment (veteran29) [inline addons/fieldmanual/stringtable.xml:311]:
```suggestion
            <Polish>Niższa Temperatura twojej Broni</Polish>
```

File: addons/fieldmanual/stringtable.xml [hunk 8]
```diff
@@ -289,27 +368,35 @@
...
+            <Polish>%3Wytrychy%4 są używane w celu uzyskania dostępu do zablokowanych pojazdów.&lt;br/&gt;&lt;br/&gt;%3Użycie:%4&lt;br/&gt;%2Wyekwipuj %3Wytrych%4.&lt;br/&gt;%2Podejdź do %3Zablokowanego%4 pojazdu. &lt;br/&gt;Użyj [%3%13%4] i wybierz %3Otwórz Zamek%4.&lt;br/&gt;&lt;br/&gt;&lt;t underline='1'&gt;%3Uwaga:%4&lt;/t&gt; Klucze i wytrychy są dostępne tylko przy zastosowaniu skryptów lub użyciu modułu ACE Zamknięcie Pojazdu</Polish>
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_VehicleKey_DisplayName">
             <English>Vehicle Keys</English>
+            <Polish>Kluczyki od Pojazdu</Polish>
         </Key>
         <Key ID="STR_ACE_FieldManual_Items_VehicleKey_ShortName">
             <English>Lock/Unlock Vehicles</English>
+            <Polish>Zablokuj/Odbloku Pojazdy</Polish>
```
Comment (veteran29) [inline addons/fieldmanual/stringtable.xml:395]:
```suggestion
            <Polish>Zablokuj/Odblokuj Pojazdy</Polish>
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

**Review by veteran29 — APPROVED** [review]

**Pull request merged**